package/MDAnalysis/coordinates/PDB.py

```diff
@@ -414,6 +418,10 @@ def _read_frame(self, frame):
                 except ValueError:
                     # Be tolerant for ill-formated or empty occupancies
                     pass
+                try:
```

Comment:
This isn't the prettiest looking code 🤣 - if it's just going to be the one addition then fair, if we envision expanding this ever then maybe we should use a nicer looking loop or something